### ISSUE TITLE
deb publish fails during s3 rm of temporary package

### ISSUE THREAD

**Issue by jessereynolds** [flapjack/omnibus-flapjack#90]
Looks like there's a missing space in this command:

aws s3 rm s3://packages.flapjack.io/tmp/flapjack_1.4.0~rc1~20150312131557~v1.4.0rc1~wheezy-1_amd64.deb--region us-east-1 2>&1

```
Finished in 2.22 seconds (files took 0.45455 seconds to load)
28 examples, 0 failures

STDERR:

Docker run completed, duration was 3m 59s.
Removing flapjack_1.4.0~rc1~20150312131557~v1.4.0rc1~wheezy-1_amd64.deb from packages.flapjack.io/tmp
rake aborted!
Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '255'
---- Begin output of aws s3 rm s3://packages.flapjack.io/tmp/flapjack_1.4.0~rc1~20150312131557~v1.4.0rc1~wheezy-1_amd64.deb--region us-east-1 2>&1 ----
STDOUT: Unknown options: us-east-1
STDERR:
---- End output of aws s3 rm s3://packages.flapjack.io/tmp/flapjack_1.4.0~rc1~20150312131557~v1.4.0rc1~wheezy-1_amd64.deb--region us-east-1 2>&1 ----
Ran aws s3 rm s3://packages.flapjack.io/tmp/flapjack_1.4.0~rc1~20150312131557~v1.4.0rc1~wheezy-1_amd64.deb--region us-east-1 2>&1 returned 255
/var/lib/gems/1.9.1/gems/mixlib-shellout-1.6.1/lib/mixlib/shellout.rb:272:in `invalid!'
/var/lib/gems/1.9.1/gems/mixlib-shellout-1.6.1/lib/mixlib/shellout.rb:259:in `error!'
/home/ubuntu/omnibus-flapjack/Rakefile:589:in `block (2 levels) in <top (required)>'
/home/ubuntu/omnibus-flapjack/Rakefile:497:in `each'
/home/ubuntu/omnibus-flapjack/Rakefile:497:in `block in <top (required)>'
Tasks: TOP => build_and_publish => test
(See full trace by running task with --trace)
Unknown options: us-east-1

I, [2015-03-12T13:35:01.763296 #10318]  INFO -- : Failed build of v1.4.0rc1 on debian wheezy at 2015-03-12T13:35:01Z, duration: 19m 5s
```

**Comment by jessereynolds** [comment]
00:05 <roberta> Failed build of Flapjack v1.4.0rc1 for debian wheezy package (experimental) in 19m 5s, log: http://is.gd/ZkCikD

**Comment by jessereynolds** [comment]
It might make sense to at least move the rm of the temporary package till after the package publish is completed?

**Comment by Hobbsee** [comment]
Yep, agreed.

**Comment by Hobbsee** [comment]
[master 882d9ec] Name docker containers so we can support multiple concurrent builds, add missing space, only remove the temporary deb after the publish occurs (closes #87, #90)
